search/searcher/search_phrase_test.go

```diff
@@ -130,7 +130,7 @@ func TestMultiPhraseSearch(t *testing.T) {
 		if err != nil {
 			t.Error(err)
 		}
-		searcher, err := NewMultiPhraseSearcher(nil, reader, test.phrase, "desc", soptions)
+		searcher, err := NewMultiPhraseSearcher(nil, reader, test.phrase, "desc", soptions, 0, 0, 1.0)
```

Comment:
I think adding a new UT to demonstrate fuzziness would be good.

Reply:
UT?